.vscode/launch.linux.json

```diff
@@ -87,6 +87,32 @@
         }
       ]
     },
+    {
+      "name": "Tests Extension",
+      "preLaunchTask": "Build Only (debug)",
+      "program": "${workspaceFolder}/extern/nvidia/_build/target-deps/kit-sdk/kit",
+      "args": [
+        "${workspaceFolder}/apps/cesium.omniverse.cpp.tests.runner.kit"
+      ],
+      "env": {
+        // Disable LSAN when debugging since it doesn't work with GDB and prints harmless but annoying warning messages
+        "ASAN_OPTIONS": "detect_leaks=0",
+        "UBSAN_OPTIONS": "print_stacktrace=1"
+      },
+      "cwd": "${workspaceFolder}",
+      "type": "cppdbg",
+      "request": "launch",
+      "MIMode": "gdb",
```

Comment:
Could you add

```
      "symbolLoadInfo": {
        "loadAll": false,
        "exceptionList": "libcesium.omniverse.plugin.so;libcesium.omniverse.cpp.tests.plugin.so"
      },
```

to this configuration? That will make it start up faster. CC https://github.com/CesiumGS/cesium-omniverse/pull/478